src/CRM/CivixBundle/Command/InitCommand.php

```diff
@@ -103,6 +107,30 @@ protected function execute(InputInterface $input, OutputInterface $output) {
       $output->writeln('<error>Unrecognized license (' . $ctx['license'] . ')</error>');
       return;
     }
+
+    if ($ctx['fullName'] !== $ctx['mainFile']) {
+      $output->writeln("");
+      $output->writeln("<info>ALERT: The requested command requires split-naming.</info>");
+      $output->writeln("");
+      $output->writeln("CiviCRM extensions formally have two names, the \"key\" and the \"file\".");
+      $output->writeln("Some APIs use the \"key\" name, and other APIs use the \"file\" name.");
+      $output->writeln("");
+      $output->writeln("   <comment>\"Key\":</comment> Appears in many strings+indices. Allows Java-style prefix.");
+      $output->writeln("         <comment>Requested Value:</comment> {$ctx['fullName']}");
+      $output->writeln("         <comment>Example Usage:</comment> addStyleFile('{$ctx['fullName']}', 'example.css')");
+      $output->writeln("   <comment>\"File\"</comment>: Appears in PHP files+functions. No Java-style prefix.");
+      $output->writeln("         <comment>Requested Value:</comment> {$ctx['mainFile']}");
+      $output->writeln("         <comment>Example Usage:</comment> function {$ctx['mainFile']}_civicrm_config() {}");
+      $output->writeln("");
+      $output->writeln("Many developers find it easier to use matching names, but the");
+      $output->writeln("requested command requires splitting the names. You may continue with");
+      $output->writeln("split names, or you may cancel and try again with a simpler name.");
+      $output->writeln("");
+      if (!$this->confirm($input, $output, "Continue with current (split) name? [Y/n] ")) {
```

Comment:
This doesn't work on windows but that's come up elsewhere so isn't something new, just is new in this command. The workaround is just don't use dots.

Reply:
Hmm, curious. I don't have a good setup to try it out. FWIW, that's  a [thin wrapper](https://github.com/totten/civix/blob/master/src/CRM/CivixBundle/Command/AbstractCommand.php#L18-L29) around [Symfony `ConfirmationQuestion`](https://symfony.com/doc/4.4/components/console/helpers/questionhelper.html). Theories:

* The command somehow gets option`--yes`.
* There's a discrepancy in the version of `symfony/console`. (Ex: My local has `v4.4.34`)
* There's an upstream issue with `ConfirmationQuestion` in the active version of  `symfony/console`.

Reply:
Ahh - symfony/console version. I must have not run composer install anytime recently.